mobile/src/main/java/org/openhab/habdroid/ui/OpenHABAboutActivity.java

```diff
@@ -248,9 +248,7 @@ private String getServerSecret(Connection conn) {
             conn.getSyncHttpClient().get("/static/secret", new MyHttpClient.TextResponseHandler() {
                 @Override
                 public void onFailure(Call call, int statusCode, Headers headers, String responseString, Throwable error) {
-                    if (error.getMessage() != null) {
-                        Log.d(TAG, error.getMessage());
-                    }
+                    Log.d(TAG, "Could not fetch server secret", error);
```

Comment:
@maniac103 Do you think we need the stacktrace here?

Reply:
No, probably not. We know the call path here anyway.